packages/docs/pages/integrating-with-namada/sdk/shielded-transfers.mdx

```diff
@@ -56,4 +61,33 @@ async fn construct_transfer(sdk: impl Namada) {
       Err(e) => println!("Tx error: {:?}", e)
     }
 }
-```
\ No newline at end of file
+```
+
+### Frontend sustainability fee
+
+For transactions that move assets into or out of the shielded pool, the namada
+SDK provides the possibility to charge an optional fee to support the operations
+of the frontend provider. The specific transactions are:
+
+- `TxShieldingTransfer`
+- `TxUnshieldingTransfer`
+- `TxOsmosisSwap`
+- `TxIbcTransfer`
+- `GenIbcShieldingTransfer`
```

Comment:
can you provide some more detail on the general flow? Are operators expected to build their own instance of Namadillo on top of some release that makes certain SDK additions? Sorry I kinda have no idea myself. I think there should be more detail on how to integrate the SDK changes into one's own hosted Namada (or let me know if this is obvious and I'm just clueless)